core/domain/exp_services.py

```diff
@@ -1828,10 +1828,15 @@ def _create_change_list_from_suggestion(suggestion):
                 object.
     """
 
-    return [{'cmd': exp_domain.CMD_EDIT_STATE_PROPERTY,
-             'state_name': suggestion.state_name,
-             'property_name': exp_domain.STATE_PROPERTY_CONTENT,
-             'new_value': [suggestion.state_content]}]
+    return [{
+        'cmd': exp_domain.CMD_EDIT_STATE_PROPERTY,
+        'state_name': suggestion.state_name,
+        'property_name': exp_domain.STATE_PROPERTY_CONTENT,
+        'new_value': {
+            'html': suggestion.suggestion_html,
+            'audio_translations': []
```

Comment:
Hmm this does seem a little bit of a conflict since I was originally thinking of doing the invalidation on the client-side, but with suggestions it seems like a server-side route is taken for updates. I suppose we could have the suggestion only update the html and then also create a back-end function that replicates deprecateAudioTranslations. However, if we're doing this it might be cleaner to just do all invalidation on the backend.